src/CameraAravis.cpp

```diff
@@ -276,21 +276,21 @@ namespace camera
 				arv_camera_set_exposure_time_auto(camera, ARV_AUTO_OFF);				
 				arv_camera_set_exposure_time(camera, value);
 				double exposure = arv_camera_get_exposure_time(camera);
-				if (exposure != value) throw runtime_error("The attribute 'exposure' could not be set");		
+				if (abs(exposure - value) > 50) throw runtime_error("The attribute 'exposure' could not be set "  + std::to_string(value) + " but the camera reports " + std::to_string(exposure));		
```

Comment:
I don't think it's a good idea. Why would '50' be good ?

I think the right way to handle it is in your camera configuration, that is: set a value that is valid for the camera.